staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller_reconcile.go

```diff
@@ -48,7 +50,11 @@ type policyController struct {
 
 	// Provided to the policy's Compile function as an injected dependency to
 	// assist with compiling its expressions to CEL
-	ValidatorCompiler
+	cel.FilterCompiler
```

Comment:
Should these extra parameters be embedded or named? b/c it's wrapping filterCompiler, I'm guessing it should be a named member?

Is anyone outside of the struct calling Compile()?

Reply:
updated